draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -2412,12 +2412,12 @@ not explicitly specifying a `SameSite` attribute being included on top-level
 cross-site requests with "unsafe" HTTP methods (as was the case prior to the
 introduction of the `SameSite` attribute).
 
-For example, a login flow may involve a cross-site top-level `POST` request to
-an endpoint which expects a cookie with login information. For such a cookie,
-"Lax" enforcement is not appropriate, as it would cause the cookie to be
-excluded due to the unsafe HTTP request method. On the other hand, "None"
-enforcement would allow the cookie to be sent with all cross-site requests,
-which may not be desirable due to the cookie's sensitive contents.
+For example, the concluding step of a login flow may involve a cross-site top-level
+`POST` request to an endpoint which expects a cookie with transactional state
+information necessary to securely complete the login. For such a cookie, "Lax"
+enforcement is not appropriate, as it would cause the cookie to be excluded due
+to the unsafe HTTP request method, resulting in an unrecoverable failure of the
+whole login flow.
 
 The "Lax-allowing-unsafe" enforcement mode described in {{lax-allowing-unsafe}}
```

Comment:
Seems github won't let me add a suggestion for this text, but I also suggest a change of

```
The "Lax-allowing-unsafe" enforcement mode described in {{lax-allowing-unsafe}}
retains some of the protections of "Lax" enforcement (as compared to "None")
while still allowing recently created cookies to be sent cross-site with unsafe
top-level requests.
```

Reply:
Yeah, github doesn't let you suggest changes too far from the context of the original changes in the PR. But I'll apply this locally and push and the whole dance. Thanks for the suggestion.

Reply:
done in 4590b6c0b949791f907eb240c59c3fe0a5a6bace